packages/server/lib/makeDataContext.ts

```diff
@@ -1,5 +1,5 @@
 import { DataContext, getCtx, clearCtx, setCtx } from '@packages/data-context'
-import type { OpenDialogOptions, SaveDialogOptions, BrowserWindow } from 'electron'
+import electron, { OpenDialogOptions, SaveDialogOptions, BrowserWindow } from 'electron'
```

Comment:
I am wondering how this is working, because we use `makeDataContext` in the tests (e2ePluginSetup & some of the mocha tests). 

Maybe v18 it handles it better?

Reply:
Hmm, this is what we were doing before: https://github.com/cypress-io/cypress/pull/21174/files#diff-f4f06c00a1bc66aafb7f77e5c5ea7b04152df534668fe536d8deb2cdbe950e67L2

Seems to be fine, but if there are concerns we can keep this as-is?

Reply:
Yeah no concerns, if the tests are passing I'm good with it